File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03703/p03703_num1_parsed.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int N, K;
    cin >> N >> K;

    vector<long long> a(N);
    for (int i = 0; i < N; i++) {
        cin >> a[i];
    }

    long long count = 0;

    for (int i = 0; i < N; i++) {
        long long sum = 0;
        for (int j = i; j < N; j++) {
            sum += a[j];
            long long mean = sum / (j - i + 1);
            if (mean >= K) {
                count++;
            }
        }
    }

    cout << count << endl;

    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03703/p03703_num1_parsed.py
'''
def count_subsequences(n, k, arr):
    count = 0
    for i in range(n):
        for j in range(i+1, n+1):
            # Calculate the sum of subsequence
            sub_sequence_sum = sum(arr[i:j])
            # Calculate the average of subsequence
            sub_sequence_avg = sub_sequence_sum / (j - i)
            
            if sub_sequence_avg >= k:
                count += 1
    
    return count

# Read input from standard input
n, k = map(int, input().split())
arr = []
for _ in range(n):
    arr.append(int(input()))

# Call the function and print the result
result = count_subsequences(n, k, arr)
print(result)
'''